Activities/DocAcquire/DocAcquire/Services/DocumentExtractionService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using DocAcquire.DataContracts;
+using DocAcquire.Helpers;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace DocAcquire
+{
+    public class DocumentExtractionService : IDocumentExtractionService
```

Comment:
having multiple service classes with one method each might be overkill. consider consolidating this in one DocAquireService with more methds

Reply:
I agree where you are coming from, but chances are these classes many change and evolve in future, so will be good to keep them seperate.